### PR TITLE
chezmoi: Add version 2.10.0

### LINKED ISSUE #3220: [Request] chezmoi
### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [x] a non-GUI tool
- [x] a reasonably well-known and widely used developer tool
- [x] the latest stable version of the program
- [x] the full version i.e. not a trial version
- [x] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** chezmoi

**Description:** Manage your dotfiles across multiple machines, securely. With first-class Windows support.

**Homepage:** https://www.chezmoi.io/

**Download link(s):** https://github.com/twpayne/chezmoi/releases

**Some indication of popularity/repute:** 5.8K stars on GitHub, the [most popular dotfile manager](https://dotfiles.github.io/utilities/)

chezmoi has been available as a Scoop bucket for two years: https://github.com/twpayne/scoop-bucket

Follow a request from @LazyGeniusMan in https://github.com/twpayne/scoop-bucket/issues/2, would you consider including this package in Main? I (the creator of chezmoi) would be happy to maintain chezmoi as a Scoop Main bucket.

### PR DESCRIPTION
**Pull request by LazyGeniusMan**
Closes #3220 https://github.com/twpayne/scoop-bucket/issues/2

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### Note
There are files in `completions` folder:
```
./completions:
chezmoi-completion.bash
chezmoi.fish
chezmoi.ps1
chezmoi.zsh
```
Is there something I have to do for those file?

### CODE REVIEW

**Comment by LazyGeniusMan** [comment]
> There are files in `completions` folder:
> 
> ```
> ./completions:
> chezmoi-completion.bash
> chezmoi.fish
> chezmoi.ps1
> chezmoi.zsh
> ```
> 
> Is there something I have to do for those file?

regarding this maybe @twpayne can help?

File: bucket/chezmoi.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "2.10.0",
+    "description": "Manage your dotfiles across multiple machines, securely. With first-class Windows support.",
```
Comment (rashil2000) [inline bucket/chezmoi.json:3]:
```suggestion
    "description": "Manage your dotfiles across multiple diverse machines, securely.",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
In autoupdate, hash property can be moved out of architecture block, since it is same (common) for both 32 and 64 bit

**Comment by rashil2000** [comment]
Scoop does not install completion files. Users need to source them themselves, depending on their favourite shell. We can leave the folder as is.

**Comment by LazyGeniusMan** [comment]
> In autoupdate, hash property can be moved out of architecture block, since it is same (common) for both 32 and 64 bit

You meant like this?
```json
"autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/twpayne/chezmoi/releases/download/v$version/chezmoi_$version_windows_amd64.zip"
            },
            "32bit": {
                "url": "https://github.com/twpayne/chezmoi/releases/download/v$version/chezmoi_$version_windows_i386.zip"
            }
        },
        "hash": {
            "url": "$baseurl/checksums.txt",
            "regex": "$sha256\\s*$basename"
        }
    }
```

> Scoop does not install completion files. Users need to source them themselves, depending on their favourite shell. We can leave the folder as is.

Any notes need to be added?

**Comment by rashil2000** [comment]
Yes.

> Any notes need to be added?

Not required

**Pull request merged**